tests/integration/route-parameter.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import 'mocha';
+import 'reflect-metadata';
+import * as request from 'supertest';
+import { iocContainer } from '../fixtures/routeParameter/ioc';
+import { RouteParameterService } from '../fixtures/routeParameter/routeParameterService';
+import { app } from '../fixtures/routeParameter/server';
+import { TestModel, TestSubModel } from '../fixtures/testModel';
+
+const basePath = '/v1';
+
+describe('RouteParameter Express Server', () => {
+  it('can handle get request with no path argument', () => {
```

Comment:
why not just add another controller to `fixtures/controllers`, but a whole new "environment" (`fixtures/routeParameter`)?